test_sd.ino

```diff
@@ -0,0 +1,61 @@
+#if false
```

Comment:
I like the test .ino file, it works. I think it would be better to just basically initialize and have some more print outs within the 'selfTest'. But this is just me. Once you comment on this, I will approve and merge

Reply:
The thing is that `selfTest` is also called from the actual capsule code. I understand the utility of print statements inside there -- and I had to temporarily add some while debugging this weekend -- but I don't want to send that information over the radio while in flight.

Reply:
thats fair. just wanted to hear back about your thoughts. i will approve.